README.md

```diff
@@ -50,8 +50,6 @@ When `REFERENCE_RELEASE_TYPE` is `1.3`, reference release contains following com
 - Change log
 - Quick start guide
 
-Note: Currently, only nRF52840 dongles are supported for CLI/RCP firmwares.
```

Comment:
Mason, Let's put this comment back and add to it.  We can say something like, `Note: Currently, only nRF52840 dongles and EFR32 4166A boards are supported for CLI/RCP firmwares.`